### ISSUE TITLE
Retraining Vgg from existing weights

### ISSUE THREAD

**Issue by saivineethkumar** [pjreddie/darknet#899]
Hi @pjreddie 
I want to retrain the vgg model for classification of more objects, that is presently we can classify 1000 classes using imagenet, but i want to increase this to something like 1005.
I tried similar steps that you listed for cifar training but could not achieve success as cifar is trained from scratch but i want to train vgg from existing weights.

**This is how i wanted to train my model:**
I want to expand the last layer of the vgg from 1000 to 1005 neurons so that the weights on the old links will be retained and the new ones will be randomized. when I start training, these weights will be altered according to GD algorithm. I will keep an eye on number of epochs to train based on some threshold value of accuracy of prediction of old classes. Now after completing the training the accuracy of the old classes will not be below a threshold and for the new ones, it will be good enough.

So my question is: Is it possible to train my model like this? Can you suggest me a way to do training like this. I dont want to start training from scratch, I want to use the existing weights as starting point for training. 

And also if we change the last layer neurons from 1000 to 1005 and we are importing the pretrained weight file which is trained for 1000 neurons, will it fit our new model ? if yes, will the other weights that are not loaded will be randomly assigned?

Thanks!
